[COST-5607] update whats new in 3.3.2 (#473)
project-koku/koku-metrics-operator: docs/csv-description.md

* PersistentVolumeClaim (PVC) configuration: The KokuMetricsConfig CR can accept a PVC definition and the operator will create and mount the PVC. If one is not provided, a default PVC will be created.
* Restricted network installation: this operator can function on a restricted network. In this mode, the operator stores the packaged reports for manual retrieval.

## New in v3.3.2:
* Leader election timing is now configurable via these environment variables:
  * `LEADER_ELECTION_LEASE_DURATION` - sets the lease timeout duration
  * `LEADER_ELECTION_RENEW_DEADLINE` - specifies the deadline to renew the lease

>   * `LEADER_ELECTION_RETRY_PERIOD` - defines the interval between retry attempts

Reviewer: So how does a customer change these? That should be documented somewhere.

Author: Thanks for catching this, updated the docs to include guidance.

* PersistentVolumeClaim (PVC) configuration: The KokuMetricsConfig CR can accept a PVC definition and the operator will create and mount the PVC. If one is not provided, a default PVC will be created.
* Restricted network installation: this operator can function on a restricted network. In this mode, the operator stores the packaged reports for manual retrieval.

## New in v3.3.2:

> * Leader election timing is now configurable via the environment variables below. To update these settings, [modify the variables in a Subscription](https://github.com/operator-framework/operator-lifecycle-manager/blob/5a01f50258003e248bd5630df0837fe0bb0f1cb7/doc/design/subscription-config.md) and apply the changes.

Reviewer: ```suggestion
* Leader election settings are now configurable via environment variables. These variables can be modified in the operator [Subscription](https://github.com/operator-framework/operator-lifecycle-manager/blob/5a01f50258003e248bd5630df0837fe0bb0f1cb7/doc/design/subscription-config.md).
```

  
  ```
      kind: Subscription
      metadata:
      ...
      spec:
        ...
        config:
          env:
            - name: LEADER_ELECTION_LEASE_DURATION  # sets the lease timeout duration
              value: "60s"
            - name: LEADER_ELECTION_RENEW_DEADLINE  # specifies the deadline to renew the lease
              value: "40s"
            - name: LEADER_ELECTION_RETRY_PERIOD    # defines the interval between retry attempts

>               value: "15s"

Reviewer: I think the variable names are obvious, so the comment doesn't provide much value. It would be good to mention that the value needs to be a duration. What format is the variable expected to be?

Author: updated to include expected format for values
